pages/pipelines/command_step.md

```diff
@@ -22,9 +22,10 @@ Required attributes:
   <tr>
     <td><code>command</code></td>
     <td>
-      The shell command/s to run during this step. This can be a single line of commands, or a list of commands that must all pass. Also available as the alias <code>commands</code>.<br>
+      The shell command/s to run during this step. This can be a single line of commands, or a list of commands that must all pass.<br>
       <em>Example:</em> <code>"build.sh"</code><br>
-      <em>Example:</em><br><code>- "npm install"</code><br><code>- "tests.sh"</code>
+      <em>Example:</em><br><code>- "npm install"</code><br><code>- "tests.sh"</code><br>
+      <em>Alias:</em>em> <code>commands</code>
```

Comment:
I'll fix this line in a subsequent commit that removes the accidentally added `em>`.

Reply:
This is why I like the `suggestion` block:

```suggestion
      <em>Alias:</em> <code>commands</code>
```